doc/DeveloperGuide.md

```diff
@@ -59,8 +59,10 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * * `| user | find a person by a tag| locate groups of people without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
 `*` | user with many persons in the address book | sort persons by name | locate a person easily
+`*` | user | undo/redo my commands | so I can reverse or change the action I wanted to do
```

Comment:
You can think of specific users and write user-stories to handle their requirements. This will help you to define the target customers for your product.